.gitignore

```diff
@@ -34,6 +34,7 @@ manifest_dump
 sst_dump
 blob_dump
 block_cache_trace_analyzer
+db_basic_test_with_timestamp
```

Comment:
You can follow the naming convention for unit test here. Renaming it to `db_with_timestamp_basic_test` will make it unnecessary for this line.